### DISCUSSION TITLE
[Help]: Is there a way to increase spring force?

### DISCUSSION THREAD

**Discussion by tarikmehinagic** [davidjerleke/embla-carousel#1006] [Help] [answered]
### Summary

Example: 
I’ve got a horizontal carousel with 6 to 8 slides in the viewport. When I drag from the leftmost slide all the way to the right, it can move about 50% of the viewport, which looks pretty ugly imo. Is there a way to increase the spring force so it doesn’t let me drag that much when I’m at the start of the carousel?

Video: https://github.com/user-attachments/assets/eff80f9c-03b0-4559-bfc0-d2fc9f0c02fb

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [X] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment, marked as answer]
> I’ve got a horizontal carousel with 6 to 8 slides in the viewport. When I drag from the leftmost slide all the way to the right, it can move about 50% of the viewport, which looks pretty ugly imo. Is there a way to increase the spring force so it doesn’t let me drag that much when I’m at the start of the carousel? 

Hi @tarikmehinagic,
Thanks for your question. Well yes, that happens on desktops, on touch devices it's not possible to drag the same distance because of natural reasons. But to answer your question: No, it's not possible at the time of writing.
